### PR TITLE
openexr/ilmbase: update to 2.5.2.

### PR DESCRIPTION
**Pull request by skmpz**
locally compiled:
- gimp: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- darktable: x86_64 aarch64 x86_64-musl aarch64-musl (archs=x86_64* aarch64* ppc64le*)
- calligra: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- PrusaSlicer: x86_64 i686 and x86_64-musl (nocross)
- alembic: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- hugin: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- luxmark: x86_64 (depends on embree)
- luxrays: x86_64 (depends on embree)
- kio-extras: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- kimageformats: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- gmic: x86_64 i686 and x86_64-musl (nocross)
- krita: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- blender: x86_64 i686 and x86_64-musl (archs="x86_64* i686* ppc64*)
- opencv4: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- opencv: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- openimageio: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- openvdb: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- synfig: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- synfigstudio: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl
- vips: x86_64 aarch64 armv7l x86_64-musl aarch64-musl armv6l-musl

### CODE REVIEW

**Comment by fosslinux** [comment]
Please:

1. Remove all of the merge commits
2. Split the `openexr/ilmbase: update to 2.5.2.` commits out into 1 commit for openexr and 1 commit for ilmbase, and place them at the beginning of the commit stack.
3. All of the miscellaneous ones at the end such as `remove patches`, `update gmic template for 2.5 libraries` and `change hugin, luxmark and luxrays patches` all need to follow the correct commit naming scheme. https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes
4. Ideally, the revbumps would show which of the two packages they are revbumping against. For example, `kio-extras` only is being revbumped for openexr.

File: srcpkgs/hugin/patches/find-openexr-2_5.patch [hunk 1]
```diff
@@ -23,7 +23,7 @@
    find_library_with_debug(OPENEXR_ILMTHREAD_LIBRARY 
      WIN32_DEBUG_POSTFIX d
 -    NAMES IlmThread-2_3 IlmThread-2_2 IlmThread
-+    NAMES IlmThread-2_4 IlmThread-2_3 IlmThread-2_2 IlmThread
++    NAMES ilmthread-2_5 ilmthread-2_4 ilmthread-2_3 IlmThread-2_2 IlmThread
```
Comment (ahesford) [inline srcpkgs/hugin/patches/find-openexr-2_5.patch:26]: Not sure if this is case sensitive, but match the `IlmThread` capitalization for consistency unless the all-lowercase form is needed to work.

**Comment by skmpz** [comment]
Ok guys thanks for the comments, I will do the changes and open a new PR. Closing this one.

**Comment by ericonr** [comment]
@skmpz you can keep working on the changes here and push force, please don't close the PR to open a new one.

**Comment by pullmoll** [comment]
Thou shalt not close existing and open new PRs addressing the same issue.
Your friendly reviewer team is fed with close and open mail notifications.

Amend or edit your commits, remove, drop, squash, clean up and finally force push the result to the existing PR.
This is [the rule](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#starting-a-pull-request), not merely a suggestion.

**Comment by skmpz** [comment]
Ok, reopened.

**Pull request closed without merging**